experiments/run_benchmark.py

```diff
@@ -129,9 +129,9 @@ def main() -> None:
     else:
         model_adapter.model.to(config.device)
 
-    train_dataset, _ = data_utils.get_dataset(args.eval_dataset)
+    train_dataset = data_utils.get_dataset(args.eval_dataset)
```

Comment:
Thanks a lot for catching this @lopusz! Could you possibly rename `train_dataset` to simply `dataset`, or instead `train_dataset = data_utils.get_dataset(args.eval_dataset)["train"]`? It's only cosmetic but makes things even clearer. Cheers